draft-ietf-mls-protocol.md

```diff
@@ -3219,6 +3229,24 @@ The second form of authentication is that group members can verify a message
 originated from a particular member of the group. This is guaranteed by a
 digital signature on each message from the sender's signature key.
 
+The signature keys held by group members are critical to the security of MLS
+against active attacks.  If a member's signature key is compromised, then an
+attacker can create KeyPackages impersonating the member; depending on the
+application, this can then allow the attacker to join the group with the
+compromised member's identity.  For example, if a group has enabled external
+parties to join via external commits, then an attacker that has compromised a
+member's signature key could use an external commit to insert themselves into
+the group -- even using a "resync"-style external commit to replace the
+compromised member in the group.
+
+Applications can mitigate the risks of signature key compromise using pre-shared
+keys.  If a group requires joiners to know a PSK in addition to authenticating
+with a credential, then in order to mount an impersonation attack, the attacker
+would need to compromise the relevant PSK as well as the victim's signature key.
+The cost of this mitigation is that the application needs some external
+arrangement that ensures that the legitimate members of the group to have the
+required PSKs.
```

Comment:
Maybe worth noting that resumption secrets can be optionally used to mitigate the signature key compromise in the "resync" scenario.